### PR TITLE
Updating the TestGrid documentation.

### PR DESCRIPTION
**Pull request by michelle192837**
Adding small summar widget, table of contents, small fixes.

### CODE REVIEW

File: testgrid/README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Testgrid
 
+### Table of Contents
+[Configuration](#configuration)
```
Comment (fejta) [inline testgrid/README.md]: Make these bullets?

File: testgrid/README.md [hunk 2]
```diff
@@ -259,6 +271,20 @@ TestGrid assigns dashboard tabs a status based on recent test runs.
  *  **FLAKY**: The tab is neither PASSING nor FAILING. There is at least one
     recent failed result that is not a consistent failure.
 
+### Summary Widget
+
+You can get a small widget showing the status of your dashboard tab, based on
+the tab statuses above! For example:
+
+sig-testing-misc#bazel: [![sig-testing-misc/bazel](https://amerai-imagefix-dot-k8s-testgrid.appspot.com/q/summary/sig-testing-misc/bazel/tests_status?style=svg)](https://k8s-testgrid.appspot.com/sig-testing-misc/bazel)
```
Comment (fejta) [inline testgrid/README.md]: Maybe `sig-testing-misc#bazel` instead of plaintext?

File: testgrid/README.md (hunk 1 above)
Comment (michelle192837) [reply testgrid/README.md]: Done

File: testgrid/README.md (hunk 2 above)
Comment (michelle192837) [reply testgrid/README.md]: Done

**Comment by fejta** [comment]
/lgtm

oh

**Pull request merged**